src/brokers/metrics/components/CardBrokerMemoryUsageMetrics/CardBrokerMemoryUsageMetrics.container.tsx

```diff
@@ -0,0 +1,87 @@
+import { FC, useState, useEffect } from 'react';
+import _ from 'lodash';
+import { CardBrokerMemoryUsageMetrics } from './CardBrokerMemoryUsageMetrics';
+import {
+  usePrometheusPoll,
+  PrometheusEndpoint,
+  PrometheusResponse,
+} from '@openshift-console/dynamic-plugin-sdk';
+import { parsePrometheusDuration } from '../../../../utils';
+import { getMaxSamplesForSpan } from '../../utils';
+
+export type CardBrokerMemoryUsageMetricsContainerProps = {
+  name: string;
+  namespace: string;
+  defaultSamples?: number;
+  timespan?: number;
+  fixedEndTime?: number;
+};
+
+type AxisDomain = [number, number];
+
+export const CardBrokerMemoryUsageMetricsContainer: FC<
+  CardBrokerMemoryUsageMetricsContainerProps
+> = ({ name, namespace = 'deafult', defaultSamples = 300, timespan }) => {
```

Comment:
Was the 'deafult' intentional, or was it a copy and paste problem from the previous or other entries?